### ISSUE TITLE
Compiling on Debian 8

### ISSUE THREAD

**Issue by dthomas** [fireice-uk/xmr-stak#633]
Please provide as much as possible information to reproduce the issue.

### Basic information
```
$ lspci | grep -i vga
01:00.0 VGA compatible controller: Advanced Micro Devices, Inc. [AMD/ATI] Tahiti PRO [Radeon HD 7950/8950 OEM / R9 280]
```

### Compile issues
  - Debian 8

The drivers for 7950 are not available in Debian 9. Hence, I am forced to use Debian 8.

Is there any way to compile the software on Debain 8? I cant find CMake 3.1 or GCC 5 on ```jessie-backports``` either

  ```
$ cmake ..
CMake Error at CMakeLists.txt:3 (cmake_minimum_required):
  CMake 3.1.0 or higher is required.  You are running version 3.0.2

  ```

**Comment by roberth1990** [comment]
https://backports.debian.org/changes/jessie-backports.html

**Comment by dthomas** [comment]
Thank you. I somehow missed that update.

However, I can't build it, since GCC 5.1 is required. I will try to compile it with ```clang```.

**Comment by VirtuBox** [comment]
Try to use the docker script available and use the version for Ubuntu 16.04

**Comment by knoxcard** [comment]
cmake . -DCUDA_ENABLE=OFF -DOpenCL_ENABLE=OFF
